easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iimpi-2021a.eb

```
@@ -28,7 +28,7 @@ dependencies = [
     ('Qt5', '5.15.2'),
 ]
 
-configopts = "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
+configopts = '-DOPENGL_INCLUDE_DIR="$EBROOTMESA/include;$EBROOTLIBGLU/include" '
```

consistency across `CGAL` easyconfigs
```suggestion
configopts = r"-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
```